getting-started-service-registry/README.adoc

```diff
@@ -0,0 +1,197 @@
+[id="chap-getting-started-service-registry"]
+= Getting Started with {product-long}
+ifdef::context[:parent-context: {context}]
+:context: getting-started-service-registry
+
+////
+START GENERATED ATTRIBUTES
```

Comment:
I think this was copied from other place. We need to regenerate it

Reply:
Yes I think so too, will commit with regenerated attributes.

Reply:
done